### PR TITLE
Supported files in detonate playbook

### PR DESCRIPTION
**Pull request by karinafishman**
added task to filter the file types that are supported by the vendor

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-9192
relates: https://jira-dc.paloaltonetworks.com/browse/CIAC-7850

## Description
Added new task for filtering supported types by Polygon.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Polygon/ReleaseNotes/1_0_11.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Detonate File - Group-IB TDS Polygon
+
+- Added new task for filtering supported types by Polygon.
```
Comment (ArikDay) [inline Packs/Polygon/ReleaseNotes/1_0_11.md:6]: Added a new task for filtering supported types by Polygon

File: Packs/Polygon/Playbooks/playbook-Detonate_File_-_Group-IB_TDS_Polygon.yml [hunk 2]
```diff
@@ -371,23 +421,121 @@ tasks:
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "20":
+    id: "20"
+    taskid: f827eaad-0371-4dd5-8567-d5a3a080a499
+    type: regular
+    task:
+      id: f827eaad-0371-4dd5-8567-d5a3a080a499
+      version: -1
+      name: set
```
Comment (ArikDay) [inline Packs/Polygon/Playbooks/playbook-Detonate_File_-_Group-IB_TDS_Polygon.yml:446]: Change the name to something more explanatory.

**Review by ArikDay — APPROVED** [review]

File: Packs/Polygon/Playbooks/playbook-Detonate_File_-_Group-IB_TDS_Polygon_README.md [hunk 3]
```diff
@@ -1,96 +1,96 @@
...
-| --- | --- | --- | --- | --- |
-| File | The file object of the file to detonate. The file is taken from the context. | None | File | Optional |
-| Interval | The duration for executing the pooling (in minutes). | 1 | - | Optional |
-| Timeout | The duration after which to stop pooling and to resume the playbook (in minutes). | 60 | - | Optional |
-| Password | The password for the uploaded file. | - | - | Optional |
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| file_id | File to detonate | File.EntryID | Optional |
+| Interval | Report requests frequency \(minutes\) | 1 | Required |
+| Timeout | Report waiting timeout \(minutes\) | 60 | Required |
```
Comment (ShirleyDenkberg) [inline Packs/Polygon/Playbooks/playbook-Detonate_File_-_Group-IB_TDS_Polygon_README.md:37]:
```suggestion
| file_id | File to detonate. | File.EntryID | Optional |
| Interval | Report requests frequency \(minutes\). | 1 | Required |
| Timeout | Report waiting timeout \(minutes\). | 60 | Required |
```

File: Packs/Polygon/ReleaseNotes/1_0_11.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Detonate File - Group-IB TDS Polygon
+
+- Added a new task for filtering supported types by Polygon.
```
Comment (ShirleyDenkberg) [inline Packs/Polygon/ReleaseNotes/1_0_11.md:6]:
```suggestion
Added a new task for filtering supported types by Polygon.
```

**Comment by ShirleyDenkberg** [comment]
@ArikDay Doc review completed.

**Comment by karinafishman** [comment]
not relevant

**Pull request closed without merging**